### ISSUE TITLE
How do I prove Facebook without enabling Facebook Platform?

### ISSUE THREAD

**Issue by Mikaela** [keybase/keybase-issues#2674]
I get to where I am asked to make a Facebook post and trying to click that link I get this request:

![turn on platform - mozilla firefox_303](https://cloud.githubusercontent.com/assets/831184/20454256/d8f62b8a-ae43-11e6-843b-3abc4603b61f.png)

I am not interested in enabling Facebook platform even for Keybase (and there are far more useful services that would use Facebook platform), so how do I proof my "ownership" of my account without letting all kinds of apps (including those that "friends" use) access whatever I have entered to Facebook?

**Comment by zQueal** [comment]
Facebook platform is a blanket term used to describe a set of API tools which third party application developers use to interface with a user's facebook account. Without enabling it, I highly doubt Keybase would be able to verify the authenticity of a post, and scrape it to ensure the proof is still online.

**Comment by jeffmcneill** [comment]
@zQueal that is not strictly true. Keybase scrapes tweets just fine, without requiring app access to a twitter account. If someone's facebook post wtih Keybase public key/link were made public, it should have no trouble finding it. 

I too don't want to turn on the platform for Facebook (and I won't). However, there are people I am connected to on Facebook who may want to be aware I am on Keybase. Therefore, some Twitter-like solution is definitely desirable, for some subset of people.

I would call this a bug, rather than an enhancement request. The bug is in requiring platform access, a fix for the but is in not requiring platform access but implementing the system in a similar way to the current Twitter account verification.

**Comment by dfilskov** [comment]
I agree completely with Jeff and Mikaela.

**Comment by imathew** [comment]
Yes, is there any reason we can't just link to a public post, as I have with other services?
